portfolio/src/componants/Projects/Projects.css

```diff
@@ -0,0 +1,236 @@
+.w2-cards {
+  display: flex;
+  width: 100%;
+  height: 100%;
+  justify-content: space-around;
+  align-items: flex-start;
+  /* border: 2px 2px solid red; */
+
+  /* background-color: red; */
+  /* background: rgb(255, 165, 0); 
+   background: linear-gradient(
+    132deg,
+    rgba(255, 165, 0, 1) 0%,
+    rgba(0, 0, 0, 1) 25%,
+    rgba(255, 165, 0, 1) 50%,
+    rgba(0, 0, 0, 1) 100%
+  ); 
+  */
+}
+.projects__div {
```

Comment:
Same here. Also you can write
```
.projects__div {
}
```

nestled:
```
.projects {
    &__div {
    }
}
```

(div also isn't a very telling name..)